solidity/truffle-examples/random-datasource/test/utils.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const path = require('path')
+const API_PATH = path.join(__dirname, '../', 'apikeys.js')
```

Comment:
```suggestion
const API_PATH = path.resolve(__dirname, '../apikeys.js')
```

Reply:
Is resolve a best practice?

Reply:
No, it's just easier here since it requires 2 params instead of three.